content/resources/tutorials/contents.lr

```diff
@@ -1,11 +1,92 @@
-title: Tutorials
----
-body:
+
+dy:
```

Comment:
Am not sure what this does, but am not a lektor master anyways xD

Reply:
I'll fix this